appveyor.yml

```diff
@@ -0,0 +1,28 @@
+version: 1.0.{build}
+
+platform: x64
+shallow_clone: true # Because the git clone takes ages due to its size
+
+image:
+- Visual Studio 2017
```

Comment:
Could explain in a comment why we use this image ?

Reply:
👍 Comment added